addons/mail/static/src/discuss/core/common/discuss_core_common_service.js

```diff
@@ -134,11 +134,9 @@ export class DiscussCoreCommon {
                 const { channel_id, last_message_id, partner_id } = payload;
                 const channel = this.store.Thread.get({ model: "discuss.channel", id: channel_id });
                 if (channel) {
-                    const seenInfo = channel.seenInfos.find(
-                        (seenInfo) => seenInfo.partner.id === partner_id
-                    );
-                    if (seenInfo) {
-                        seenInfo.lastFetchedMessage = { id: last_message_id };
+                    const member = channel.channelMembers.find((m) => m.persona.id === partner_id);
+                    if (member) {
+                        member.fetched_message_id = { id: last_message_id };
```

Comment:
send id in payload, this should just become insert on channel member and not a specific notif